src/Stryker.Core/Stryker.Core.UnitTest/Mutants/CsharpMutantOrchestratorTests.cs

```diff
@@ -1948,4 +1951,315 @@ public void ShouldIncrementMutantCountUniquely()
 
         secondMutant.Id.ShouldBe(firstMutant.Id + 1);
     }
+
+    [TestMethod]
+    public void ShouldMutateCollectionExpressionSpanProperty()
+    {
+        var source = "static ReadOnlySpan<int> Value => [1, 2, 3];";
+
+        var expected =
+            "static ReadOnlySpan<int> Value => (StrykerNamespace.MutantControl.IsActive(0)?[]:[1,2,3]);";
+        ShouldMutateSourceInClassToExpected(source, expected);
+    }
+
+    [TestMethod]
+    public void ShouldMutateCollectionExpressionLocalsInMethod()
+    {
+        var source = """
+                     public void M() {
+                         int[] abc = { 5, 5 };
+                         int[] bcd = [1, .. abc, 3];
+                     }
+                     """;
+
+        var expected =
+            """
+            public void M() {
+                if (StrykerNamespace.MutantControl.IsActive(0)) { }
+                else{
+                    if(StrykerNamespace.MutantControl.IsActive(1)){
+                        int[] abc = {};
+                        int[] bcd = [1, .. abc, 3];
+                    } else {
+                        int[] abc = { 5, 5 };
+                        int[] bcd = (StrykerNamespace.MutantControl.IsActive(2)?[]:[1, .. abc, 3]);
+                    }
+                }
+            }
+            """;
+        ShouldMutateSourceInClassToExpected(source, expected);
+    }
+
+    [TestMethod]
+    public void ShouldMutateImplicitCollectionExpressionInMethod()
+    {
+        var source = """
+                     public void M() {
+                         int[] abc = { 5, 5 };
+                         var bcd = (int[])[1, .. abc, 3];
+                     }
+                     """;
+
+        var expected =
+            """
+            public void M() {
+              if (StrykerNamespace.MutantControl.IsActive(0)) {
+              } else {
+                if (StrykerNamespace.MutantControl.IsActive(1)) {
+                  int[] abc = {};
+                  var bcd = (int[])[1, ..abc, 3];
+                } else {
+                  int[] abc = {5, 5};
+                  var bcd = (int[])(
+                      StrykerNamespace.MutantControl.IsActive(2) ? [] : [ 1, ..abc, 3 ]);
+                }
+              }
+            }
+            """;
+        ShouldMutateSourceInClassToExpected(source, expected);
+    }
+
+    [TestMethod]
+    public void ShouldMutateUsedCollectionExpression()
+    {
+        var source = """
+                     public void M() {
+                         // Stryker disable String : Not mutation under test
+                         Span<string> weekDays = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
+                         foreach (var day in weekDays)
```

Comment:
I think we should strive to keep these tests as simple and clear as possible. In this case I think we can remove the foreach to only test the collection expression